frontend/packages/process-editor/src/components/Canvas/BPMNViewer/BPMNViewer.test.tsx

```diff
@@ -1,44 +1,19 @@
 import React from 'react';
 import { render, screen, act } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { BPMNViewer, BPMNViewerProps } from './BPMNViewer';
-import { textMock } from '../../../../../../testing/mocks/i18nMock';
-
-const mockAppLibVersion8: string = '8.0.1';
-const mockAppLibVersion7: string = '7.0.1';
-
-const defaultProps: BPMNViewerProps = {
-  appLibVersion: mockAppLibVersion8,
-};
+import { BPMNViewer } from './BPMNViewer';
 
 describe('Viewer', () => {
   afterEach(jest.clearAllMocks);
```

Comment:
This line can now be removed since there is no mocks within this test anymore. 😊